.github/workflows/visualization.yml

```diff
@@ -0,0 +1,37 @@
+name: visualization
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+      - 'visualization/**'
+      - '.github/workflows/visualization.yml'
```

Comment:
```suggestion
    paths:
      - 'visualization/**'
      - '.github/workflows/visualization.yml'
```

Reply:
Failing otherwise https://github.com/optuna/optuna-examples/actions/runs/920318944

```
<div class="application-main " data-commit-hovercards-enabled="" data-discussion-hovercards-enabled="" data-issue-and-pr-hovercards-enabled="" style="box-sizing: border-box; color: rgb(173, 186, 199); font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div itemscope="" itemtype="http://schema.org/SoftwareSourceCode" class="" style="box-sizing: border-box;"><main style="box-sizing: border-box; display: block;"><div class="container-xl clearfix new-discussion-timeline px-3 px-md-4 px-lg-5" style="box-sizing: border-box; margin-right: auto; margin-left: auto; max-width: none; padding: 0px !important; width: 1906px;"><div id="repo-content-pjax-container" class="repository-content " style="box-sizing: border-box; width: 1906px;"><div class="px-3" style="box-sizing: border-box; padding-left: 16px !important; padding-right: 16px !important;"><div class="d-flex flex-column flex-md-row" style="box-sizing: border-box; flex-direction: row !important; display: flex !important;"><div class="flex-auto ml-md-4 min-width-0 " style="box-sizing: border-box; flex: 1 1 auto !important; min-width: 0px !important; margin-left: 24px !important;"><div class="flex-auto pb-5" style="box-sizing: border-box; flex: 1 1 auto !important; padding-bottom: 32px !important;"><div class="js-updatable-content js-socket-channel" data-channel="eyJjIjoiY2hlY2tfc3VpdGVzOjI5NDcxMzMyNjciLCJ0IjoxNjIzMjA3OTI1fQ==--69da0a1f03ead24cff4c94304b399ac58c2eadbf420240033b3f297e3b2fc296" data-url="/optuna/optuna-examples/actions/runs/920318944/annotations_partial" style="box-sizing: border-box;"><div class="actions-fullwidth-module color-bg-primary color-shadow-small mb-3 pt-3 pb-1 border rounded" style="box-sizing: border-box; border: 1px solid var(--color-border-primary) !important; border-radius: 6px !important; box-shadow: var(--color-shadow-small) !important; background-color: var(--color-bg-primary) !important; margin-bottom: 16px !important; padding-bottom: 4px !important; padding-top: 16px !important;">
Invalid type for `on`
--


</div></div><div id="artifacts" class="js-updatable-content js-socket-channel" data-channel="eyJjIjoid29ya2Zsb3dfcnVuOjkyMDMxODk0NDphcnRpZmFjdHMiLCJ0IjoxNjIzMjA3OTI1fQ==--b0013ddbb9753dd7050a2cac5e1e61caea5f99b8ddc9753c5164d5e14a0e2b77" data-url="/optuna/optuna-examples/actions/runs/920318944/artifacts_partial" style="box-sizing: border-box;"></div></div></div></div></div></div></div></main></div></div><br class="Apple-interchange-newline">
```